rs/nervous_system/integration_tests/tests/advance_target_version.rs

```diff
@@ -11,10 +11,39 @@ use ic_sns_governance::{
 };
 use ic_sns_swap::pb::v1::Lifecycle;
 use ic_sns_wasm::pb::v1::SnsCanisterType;
+use pocket_ic::nonblocking::PocketIc;
 use pocket_ic::PocketIcBuilder;
 use std::time::Duration;
 
-const TICKS_PER_TASK: u64 = 3;
+async fn await_with_timeout<'a, T, F, Fut>(
+    pocket_ic: &'a PocketIc,
+    timeout_seconds: u64,
```

Comment:
seconds seems wrong here, given that the advance of time internal to this function is an implementation detail...

However, if you add some documentation to the method to specify 1 second and 1 tick go together, maybe it's fine.

Reply:
Regarding 1 second per round. I agree this should be documented.

Note that this function isn't specific to SNS Governance. If it were, we'd probably be able to advance 10 seconds between ticks. But having 1 second seems not to be a problem in practice, and it makes this more canister-agnostic (in case we want to use it in other tests, eventually).

Reply:
I added some documentation that 1 second = 1 tick